tests/integration/telemetry/envoyfilter/customizemetrics/customize_metrics_test.go

```diff
@@ -254,10 +254,7 @@ func setupConfig(_ resource.Context, cfg *istio.Config) {
 }
 
 func setupWasmExtension(ctx resource.Context) error {
-	proxySHA, err := env.ReadProxySHA()
-	if err != nil {
-		return err
-	}
+	proxySHA := "359dcd3a19f109c50e97517fe6b1e2676e870c4d"
```

Comment:
What are the benefits of using a hard-coded SHA?

Reply:
we can move out of istio/proxy.

Reply:
Will this hard-coded SHA be updated in future? If so, any guide/doc to guide/remind a release manager to update it?

Reply:
This is a Wasm plugin that was used as an example. We support the ABI, but don't have to support independently released Wasm artifacts. In fact, we shouldn't have to release Wasm artifacts at the same cadence since the ABI is stable across versions.